### ISSUE TITLE
WordPress.com Plans page update required

### ISSUE THREAD

**Issue by reginabally** [wordpress-mobile/WordPress-Android#14374]
The WordPress.com Plans page on WPAndroid app (My Site > Plan) is showing outdated information. The WordPress.com Blogger plan is no longer available but it appeared twice on this page.

![Screenshot_2021-02-02-11-24-35-44_1cdbe7dded7ec259ed1024b4ff1ae8db](https://user-images.githubusercontent.com/40906847/113106261-249b1f80-9235-11eb-87d2-991e79dd87a8.jpg)

On the other hand, the WordPress.com Personal plan mentioned live chat support but it's no longer a feature for this plan: https://wordpress.com/pricing/

**Comment by thehenrybyrd** [comment]
Thanks for the report @reginabally!
I do see the Personal plan listing live chat, however I wasn't able to reproduce what you're seeing with the Blogger plan. I don't see Blogger listed at all, only Free, Personal, Premium, Business, and E-commerce.
Are you seeing Blogger listed on all your sites on android? If it's only some sites, what's common between them?

**Comment by reginabally** [comment]
I see WordPress.com Blogger plan on all WordPress.com sites (Free, Personal, Premium, Business, and eCommerce). I've uninstalled and re-installed the app, I'm still seeing the WordPress.com Blogger plan appears twice on the Plans page.

**Comment by thehenrybyrd** [comment]
Thanks @reginabally! 
I was able to reproduce what you're seeing on my a12 account, but not on a non-a12 test account.

Here's what I found:
On my a12 account, with a Blogger plan on one of my sites, I saw the Blogger Plan listed **twice** in Plans on all sites, like you're seeing.
On my a12 account, after removing Blogger plan from all sites, I saw the Blogger Plan listed in Plans **only once**.
On my non-a12 account, with no Blogger plan, I did **not** see the Blogger Plan listed in Plans.
On my non-a12 account, after adding a Blogger plan on one site, I saw the Blogger Plan listed in Plans **only once**.

Based on that, it seems the Blogger Plan is listed for all a12 accounts whether they have the plan or not, so for a12s who do have the Blogger plan, it's being shown twice. I don't think we'll see users run into this, but please let me know if you get any reports.

I've added a separate issue for the Personal plan listing live chat, as that does appear to be visible to users.

**Comment by reginabally** [comment]
Thanks, @thehenrybyrd! I wasn't seeing the WordPress.com Blogger plan on my non-a12 test account so all good. 👍🏼 And thanks for creating the issue for the WordPress.com Personal plan description.

**Comment by thehenrybyrd** [comment]
The WordPress.com Plans screen has been removed from the WordPress app. 
I'll close this issue, as it no longer applies.
